run.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash 
 JAVA_TARGET_DIR="$(cd "$(dirname $0)" && pwd)/target"
-JAVA_IMAGE=registry.gitlab.com/clarin-eric/docker-alpine-supervisor-java-base:openjdk8-1.2.5
+JAVA_IMAGE=registry.gitlab.com/clarin-eric/docker-alpine-supervisor-java-base:openjdk11-1.2.5
```

Comment:
This could be bumped to `openjdk11-1.2.12` (see https://gitlab.com/CLARIN-ERIC/docker-alpine-supervisor-java-base/-/tags)